src/DGtal/topology/VoxelComplex.h

```diff
@@ -240,7 +242,7 @@ class VoxelComplex : public CubicalComplex<TKSpace, TCellContainer> {
      * Insert cell (voxel) in the khalimsky space AND in the object set.
      *
      * @param kcell input voxel
-     * @param close_it if true, apply @ref voxelClose.
+     * @param close_it if true, apply @voxelClose.
```

Comment:
why did you remove the @ref ? Looks like it would produce Doxygen errors..

Reply:
Not working apparently. You should keep the @ref.

<img width="512" alt="capture d ecran 2019-01-05 a 17 33 22" src="https://user-images.githubusercontent.com/700165/50726618-01de3700-1110-11e9-9ecf-9b4d8f13e134.png">

Reply:
Thanks, I will fix it.

Reply:
@ref's are back, thanks

Reply:
oops my bad.. You forced-pushed :)
Many thanks for this great PR !
merging